examples/benchmark/config.yml

```diff
@@ -0,0 +1,61 @@
+runtime:
+  data_dir: ../datasets/
+  outfile_dir: ./results
+  db_slug: ltdb
+  detectors:
+    - mne
+    - wfdb_xqrs
+    - pyecg_pan_tompkins
+    - biosppy_hamilton
+    - heartpy
+    - neurokit2_nk
+    - neurokit2_kalidas2017
+    - sleepecg-c
+    - sleepecg-numba
+    - sleepecg-python
```

Comment:
Maybe we can use dashes in all detectors (instead of underscores) to be consistent?